cmd/deploy.go

```diff
@@ -75,12 +76,9 @@ func (c *DeployCommand) Run(args []string) int {
 
 	extraVars := fmt.Sprintf("\"%s\"", strings.Join(vars, " "))
```

Comment:
Looks like this line is the problem... the first part isn't even necessary?

```go
extraVars := strings.Join(vars, " ")
```

Not sure what led me to that originally and now it doesn't work? 🤷‍♂